python/example/berry/berry_kevlar.md

```diff
@@ -0,0 +1,430 @@
+---
```

Comment:
For now, I wouldn't recommend reviewing this notebook. It's just my scratch/workspace for figuring out all the ugly details of Berry/INLA/kevlar. I'll clean it up down the line. Or maybe move it to the research folder and replace it with something nicer walking straightforwardly through the analysis.

Reply:
Could you just move it to `research/` now? It sounds like research!

Reply:
I can do that. Ultimately, I'd like to evolve it into the a full walk through and presentation of the Berry + INLA example. That end result does seem like something that should go in this `example` folder. But, the current version hasn't gotten there yet because things are still moving fast and changing a lot.

Reply:
I'm going to leave it here for now unless I have a hard objection.

Reply:
No need to move it. But a final thought: who is the audience for this? If pharma, maybe a powerpoint or academic/latex writeup would be better than a markdown. If open source people looking at the repo, then yes, markdown is good.